### PR TITLE
Fix for sign epoch intermittent difference 

### LINKED ISSUE #3068: Intermittent difference in the epoch date
There are intermittent difference in last two digits of the receipt number from the time the user signs and what is actually stored in the cassandra db

### PR DESCRIPTION
**Pull request by BarakStout**
Closes #3068

### CODE REVIEW

File: hmda/src/main/scala/hmda/persistence/submission/HmdaProcessingUtils.scala [hunk 1]
```diff
@@ -53,26 +53,31 @@ object HmdaProcessingUtils {
...
 
+    val submissionManager =
+      sharding.entityRefFor(SubmissionManager.typeKey, s"${SubmissionManager.name}-$submissionId")
+
     val fSubmission: Future[Option[Submission]] = submissionPersistence ? (ref => GetSubmission(ref))
 
     for {
-      s <- fSubmission
-      m = s
+      m <- fSubmission
```
Comment (PatrickHSI) [inline hmda/src/main/scala/hmda/persistence/submission/HmdaProcessingUtils.scala]: Since we're making changes here can we update to better variable names?

Comment (BarakStout) [reply hmda/src/main/scala/hmda/persistence/submission/HmdaProcessingUtils.scala]: Good point

File: hmda/src/main/scala/hmda/persistence/submission/HmdaValidationError.scala [hunk 2]
```diff
@@ -283,9 +283,8 @@ object HmdaValidationError
                 .noEditsFound()) {
             Effect.persist(signed).thenRun { _ =>
               log.info(s"Submission $submissionId signed at ${Instant.ofEpochMilli(timestamp)}")
-              updateSubmissionStatus(sharding, submissionId, Signed, log)
-              updateSubmissionReceipt(sharding, submissionId, signed.timestamp, s"${signed.submissionId}-${signed.timestamp}", log)
-              publishSignEvent(submissionId).map(signed => log.info(s"Published signed event for $submissionId"))
+              updateSubmissionStatusAndReceipt(sharding, submissionId, timestamp, s"${signed.submissionId}-${signed.timestamp}", Signed, log)
+              publishSignEvent(submissionId).map(signed => log.info(s"Published signed event for $timestamp"))
```
Comment (PatrickHSI) [inline hmda/src/main/scala/hmda/persistence/submission/HmdaValidationError.scala]: Why was the log for the signed event publishing changed from including the  signed.submissionID and signed.timestamp to just the timestamp?

Comment (BarakStout) [reply hmda/src/main/scala/hmda/persistence/submission/HmdaValidationError.scala]: unintended consequence of debugging

**Pull request merged**